swap_meet/item.py

```diff
@@ -1,2 +1,27 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+        self.condition = condition        
+
+    
+    def get_category(self):
+        return "Item"
+    
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
+
+    def condition_description(self):
+        if self.condition == 0:
+            return "heavily used"
+        elif 0 < self.condition < 4:
+            return "mint"
+        elif 4 <= self.condition <= 5:
+            return "excellent"
```

Comment:
There are some redundancies here, you can remove the `0 <` part of the second conditional since it is already handled by the first conditional. Ans the same for the final conditional. The `4 <=` can be removed since it is already handled by the preceding conditional. 
```suggestion
        if self.condition == 0:
            return "heavily used"
        elif self.condition < 4:
            return "mint"
        elif self.condition <= 5:
            return "excellent"
```